packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -596,3 +762,34 @@ body {
     display: none !important; // TODO: fix specificity issue
   }
 }
+
+//----------------
+// 10. Gradients
```

Comment:
How do we use these? They look like they're meant to be reusable utilities, but seem quite specific to the use case of the bottom of the search results.

Reply:
We introduced them because couldn't find a CSS way to allow this gradient without panel footer (like in the demo) and with a panel footer (from the Layout API, similar to Search2).

Usage:

```tsx
autocomplete({
  // ...
  render({ sections }, root) {
    render(
      <Fragment>
        <div className="aa-PanelLayout-GradientTop"></div>
        <div className="aa-PanelLayout">{sections}</div>
      </Fragment>,
      root
    );
  },
});
```